meson.build

```diff
@@ -7,7 +7,7 @@
 # - reset micro version to zero when VA-API major or minor version is changed
 project(
   'libva', 'c',
-  version : '2.9.0.0',
+  version : '2.9.0',
```

Comment:
can you bump the version to become 2.9.1? Just removing ending .0 without bumping the version might cause issues with package managers on version check (which package has greater version 2.9.0 vs. 2.9.0.0).